packages/core/src/middleware/koa-spa-proxy.ts

```diff
@@ -7,12 +7,17 @@ import type { IRouterParamContext } from 'koa-router';
 
 import { EnvSet } from '#src/env-set/index.js';
 import serveStatic from '#src/middleware/koa-serve-static.js';
+import type Queries from '#src/tenants/Queries.js';
 
+import serveCustomUiAssets from './koa-serve-custom-ui-assets.js';
+
+// eslint-disable-next-line max-params
 export default function koaSpaProxy<StateT, ContextT extends IRouterParamContext, ResponseBodyT>(
   mountedApps: string[],
   packagePath = 'experience',
   port = 5001,
-  prefix = ''
+  prefix = '',
+  queries?: Queries
```

Comment:
nit: too many arguments, can be refactored to an object.

Reply:
Refactor.